R/board_azure.R

```diff
@@ -226,13 +245,11 @@ azure_ls <- function(board, dir = "") {
   paths <- AzureStor::list_storage_files(
     board$container,
     dir = dir,
-    recursive = FALSE,
-    info = "name"
+    recursive = FALSE
```

Comment:
Why did this change?

Reply:
Because we need to know `isdir`, not only the name; see line 250 ⤵️